drs/drs-assembly/subscribe-model.conf

```diff
@@ -324,14 +149,6 @@ IRIS, TCS, ESW, NFIRASO, AOESW and NSCU to be associated with science data
   usage = "AO TT loop gain. FITS=AOTT2GAIN"
```

Comment:
Things like RTC loop gains (as well as many other parameters) are sent to the RTC from the RPG. I think it would make sense for the RTC to publish an event indicating the RPG commands that it has received. This will help both in terms of information for RTC GUIs, but also these IRIS FITS headers. Something to talk to RTC team about.